### PR TITLE
rubocop: Fix Style cops

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: lib/beaker/hypervisor/vagrant.rb [hunk 1]
```diff
@@ -135,18 +135,16 @@ def make_vfile hosts, options = {}
 
       # In case this is called directly
       FileUtils.mkdir_p(@vagrant_path)
-      File.open(@vagrant_file, 'w') do |f|
-        f.write(v_file)
-      end
+      File.write(@vagrant_file, v_file)
```
[the remark this answers is not in the document]
Comment (jay7x) [reply lib/beaker/hypervisor/vagrant.rb:138]: Answering to myself: yes, it seems it does

File: Rakefile [hunk 2]
```diff
@@ -90,32 +88,30 @@ namespace :docs do
...
     output = `bundle exec yard doc`
     puts output
-    if output =~ /\[warn\]|\[error\]/
-      fail "Errors/Warnings during yard documentation generation"
-    end
+    raise 'Errors/Warnings during yard documentation generation' if output =~ /\[warn\]|\[error\]/
     Dir.chdir( original_dir )
   end
 
   desc 'Run the documentation server in the background, alias `bg`'
```
[the remark this answers is not in the document]
Comment (bastelfreak) [reply Rakefile]: dropped it in #64

File: lib/beaker/hypervisor/vagrant.rb [hunk 3]
```diff
@@ -20,33 +20,33 @@ def randip(hypervisor=nil)
...
+          "    v.vm.provision 'shell', :path => '#{provisioner_config['path']}', :args => '#{provisioner_config['args']}' \n"
         end
-        shell_provisioner_string
-      else
-        raise "No path defined for shell_provisioner or path empty"
-      end
+        
+      
+        
+
```
[the remark this answers is not in the document]
Comment (bastelfreak) [reply lib/beaker/hypervisor/vagrant.rb]: ah ups fixed

File: lib/beaker/hypervisor/vagrant.rb [hunk 4]
```diff
@@ -7,7 +7,7 @@ class Vagrant < Beaker::Hypervisor
     require 'beaker/hypervisor/vagrant_virtualbox'
 
     def rand_chunk
-      (2 + rand(252)).to_s #don't want a 0, 1, or a 255
+      rand(2..253).to_s #don't want a 0, 1, or a 255
     end
```
[the remark this answers is not in the document]
Comment (bastelfreak) [reply lib/beaker/hypervisor/vagrant.rb:11]: indeed, fixed it.

**Comment by bastelfreak** [comment]
@jay7x should be good to go now

**Review by jay7x — APPROVED** [review]

**Pull request merged**